x11-misc/mate-notification-daemon/mate-notification-daemon-1.28.1.ebuild

```diff
@@ -16,7 +16,7 @@ DESCRIPTION="MATE Notification daemon"
 LICENSE="GPL-2+ GPL-3+"
 SLOT="0"
 
-IUSE="nls X wayland"
+IUSE="nls +X wayland"
```

Comment:
I mentioned on IRC that this is not necessary and basically causes issues.

Reply:
I apologize, I missed this message in IRC. I removed this now.